src/Form/Type/EasyAdminAutocompleteCreateType.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
It is way too much code in order to add a **create new** button. I do not want to maintain so much code to duplicate existing `autocomplete` and just add an option. Please consider adding **this feature as an option to existing autocomplete** and as new form we have to handle from scratch.